src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,45 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  currentTodo: Todo | null;
+  setEye: (result: boolean) => void;
+  setShow: (value: boolean) => void;
+}
+
+export const TodoModal: React.FC<Props> = ({
+  currentTodo,
+  setEye,
+  setShow,
+}) => {
+  const [loader, setLoader] = useState(true);
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const userFromApi = currentTodo && (await getUser(currentTodo.userId));
+
+        setUser(userFromApi);
+      } finally {
+        setLoader(false);
+      }
+    })();
+  }, []);
+
+  const onHandleClick = () => {
```

Comment:
```suggestion
  const handleClose = () => {
```
'Close' will describe it better

Also, use `onClick` in function name or `handleClick`
Usually we use `onClick` for prop name (if you get this function from props). The function itself usually called `handleClick` in such cases